tests/resources/ci/scripts/setup.sh

```diff
@@ -2,64 +2,104 @@
 
 set -Eexo pipefail
 
+# Operating system.
+OS=$(uname -s)
+
 # Semeru JDK version control constants.
 # Example:
-# Version: 11.0.14.1 
-# OpenJDK 11.0.14.1+1
-# OpenJ9 0.30.1
+# Version (SEMERU_OPEN_JDK_BUILD): 11.0.14.1 
+# OpenJDK (SEMERU_OPEN_JDK_BUILD + SEMERU_OPEN_JDK_BUILD): 11.0.14.1+1
+# OpenJ9  (SEMERU_OPENJ9_VERSION): 0.30.1
 SEMERU_OPEN_JDK_MAJOR=11
 SEMERU_OPEN_JDK_VERSION="${SEMERU_OPEN_JDK_MAJOR}.0.14.1"
 SEMERU_OPEN_JDK_BUILD=1
 SEMERU_OPENJ9_VERSION=0.30.1
-SEMERU_ARCHIVE_SHA256=25f3a8475b1f0b0ef54ff0247c7839fa4d6e7363adc2956d383a981aaa491b70
+
+SEMERU_ARCHIVE_MAC_SHA256=215e1ff6fa821309548253653e74025d6a830180abe6001db7717fde4eb991d9
+SEMERU_ARCHIVE_LINUX_SHA256=25f3a8475b1f0b0ef54ff0247c7839fa4d6e7363adc2956d383a981aaa491b70
+SEMERU_ARCHIVE_WINDOWS_SHA256=568e968f12824d65cafaf1ec99f5fc040f37097bc1a03590c8ac31877c2a6a41
 
 # Maven version control constants.
 MAVEN_VERSION=3.8.5
-MAVEN_ARCHIVE_SHA512=89ab8ece99292476447ef6a6800d9842bbb60787b9b8a45c103aa61d2f205a971d8c3ddfb8b03e514455b4173602bd015e82958c0b3ddc1728a57126f773c743
+MAVEN_ARCHIVE_SHA512=f9f838b4adaf23db0204a6cafa52bf1125bd2d649fd676843fd05e82866b596ec19c4f3de60d5e3ff17f10a63d96c141311ff9bc2bfa816eade7a5cbff2bd925
```

Comment:
You don't have to answer.. but wondering why this changed.   A bit surprised Maven would rebuild releases.  Oh well.

Reply:
Is not that Maven rebuilt a release or anything nefariours. You missed the change where in order to standardize things a bit with Maven for windows and the other platforms, instead of using tar.gz archive, i updated the code to use a zip archive since it is available. So the new SHA is for the zip archive.